### PR TITLE
docs: add greptimedb_vs_influxdb_manual.md

### PR DESCRIPTION
**Pull request by killme2008**
Please take a look @evenyag 
The report link will be updated when it's published.

### CODE REVIEW

File: docs/greptimedb_vs_influxdb_manual.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+```
+
+Run GreptimeDB with the following command, specifying the data directory via command-line parameters:
+```sh
+./greptime-linux-amd64-v0.9.1/greptime standalone start --data-home /path/to/greptime/data
+```
+
+Alternatively, you can configure additional parameters via a configuration file. Assuming the configuration file is named `config.toml`, you can start it with:
+```sh
+./greptime-linux-amd64-v0.9.1/greptime standalone start --config-file /path/to/config.toml
```
Comment (evenyag) [inline docs/greptimedb_vs_influxdb_manual.md:94]: Do we need to replace `v0.9.1` with something like `${VERSION}`?

File: docs/greptimedb_vs_influxdb_manual.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+
+[storage]
+data_home = "/home/ubuntu/greptime/data-s3"
+type = "S3"
+bucket = "test-bucket"
+root = "tsbs-data-write"
+access_key_id = "****"
+secret_access_key = "****"
+endpoint = "https://s3-endpoint/"
+region = "ap-southeast-1"
```
Comment (evenyag) [inline docs/greptimedb_vs_influxdb_manual.md]:
```suggestion
bucket = "your-bucket"
root = "tsbs-data-s3"
access_key_id = "****"
secret_access_key = "****"
endpoint = "https://s3-endpoint/"
region = "your-region"
```

File: docs/greptimedb_vs_influxdb_manual.md [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+[logging]
+dir = "/home/ubuntu/greptime/logs"
+
+[storage]
+data_home = "/home/ubuntu/greptime/data-local"
+```
+
+Startup command assuming the GreptimeDB binary path is `./greptime-linux-amd64-v0.9.1/greptime` and the config file path is `/home/ubuntu/greptime/config-local.toml`:
+```sh
+./greptime-linux-amd64-v0.9.1/greptime standalone start --config /home/ubuntu/greptime/config-local.toml
```
Comment (evenyag) [inline docs/greptimedb_vs_influxdb_manual.md]: Maybe we can assume the path as `./greptime` so it works with different versions.
```suggestion
Startup command assuming the GreptimeDB binary path is `./greptime` and the config file path is `/home/ubuntu/greptime/config-local.toml`:
```sh
./greptime standalone start --config /home/ubuntu/greptime/config-local.toml
```

File: docs/greptimedb_vs_influxdb_manual.md [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+
+[[region_engine]]
+[region_engine.mito]
+enable_experimental_write_cache = true
+experimental_write_cache_size = "20G"
+```
+
+Startup command assuming the GreptimeDB binary path is `./greptime-linux-amd64-v0.9.1/greptime` and the config file path is `/home/ubuntu/greptime/config-s3.toml`:
+```sh
+./greptime-linux-amd64-v0.9.1/greptime standalone start --config /home/ubuntu/greptime/config-s3.toml
```
Comment (evenyag) [inline docs/greptimedb_vs_influxdb_manual.md]:
```suggestion
Startup command assuming the GreptimeDB binary path is `./greptime` and the config file path is `/home/ubuntu/greptime/config-s3.toml`:
```sh
./greptime standalone start --config /home/ubuntu/greptime/config-s3.toml
```

File: docs/greptimedb_vs_influxdb_manual.md (hunk 1 above)
Comment (killme2008) [reply docs/greptimedb_vs_influxdb_manual.md:94]: Let's keep it this time because we want users to run the test directly after reading the report.

**Review by evenyag — APPROVED** [review]

**Pull request merged**